docs/references/strategies/languages/gradle/gradle.md

```diff
@@ -282,6 +283,36 @@ We classify following configurations, and dependencies originating from it as a
 - kotlinNativeCompilerPluginClasspath
 ```
 
+## Only analyzing specific sub project
+
+If you have gradle project which has one or more subprojects, you may
+only want to analyze a specific set of subprojects in some cases.
```

Comment:
If I want to scan multiple subprojects how do I specify them on the command line? Are they space separated? Comma separated?